infra/integration-image/package.list

```diff
@@ -0,0 +1,274 @@
+Desired=Unknown/Install/Remove/Purge/Hold

```

Comment:
The package list of what's in the build image shouldn't be a manually generated file checked into the repo - while this could be a useful reference for intent it doesn't capture what was actually used - it should be a build artifact generated each run.